api_calls.py

```diff
@@ -160,8 +158,8 @@ def create_part(self, course_label, book_label, number, title, label):
     def create_chapter(self, course_label, book_label, part_num, chapter_num, title, label):
         course = Course(self.client, course_label)
         book = Book(course, book_label)
-        if not book.is_booklet and part_num is None:
-            raise APIError("--part must be set. {} is not a booklet.".format(book.label))
+        if part_num is None:
```

Comment:
Might be better to use the first part in this case.  So we are uploading a booklet, there is no need to specify parts at all.